system/services/admin/DataManagerService.cfc

```diff
@@ -612,7 +612,7 @@ component {
 		if (args.orderBy is 'label') {
 			args.orderBy = labelField;
 		}
-		var idField            = _getPresideOBjectService().getIdField( arguments.objectName );
+		var idField            = arguments.idField ?: _getPresideOBjectService().getIdField( arguments.objectName );
```

Comment:
* `idField` should be declared in the list of arguments to this function (and can default to an empty string)
* in `system/handlers/admin/DataManager.cfc#900`, the value of this argument is passed in as `""` in some scenarios. We need to defend against an empty IDField here. So:

```
var idField = Len( Trim( arguments.idField ) ) ? arguments.idField : _getPresideOBjectService().getIdField( arguments.objectName );
```

Reply:
Added idField to argument list and updated code as suggested. Thanks!